targets/csharp/source/source/PlayFabGameServerSDK.cs

```diff
@@ -0,0 +1,227 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PlayFab
+{
+    // The states a Game server can get into during its lifecycle
+    public enum GameState
+    {
+        // A state has not yet been set
+        Invalid,
+
+        // The game server is starting
+        Initializing,
+
+        // The game server has finished initialization and called ReadyForPlayers
+        StandingBy,
+
+        // The game server has been allocated, players are about to connect
+        Active,
+
+        // The game server is shutting down
+        Terminating,
+
+        // The game server has ended
+        Terminated,
+
+        // PlayFab is requesting this game server remain idle for debugging purposes.
+        // Note that this state is not yet being used
```

Comment:
Is that still true? When is it supposed to start being used?

Reply:
still true, I'm not sure when we'll implement Quarantined,  it's a question for Chris I guess :) 

---
In reply to: [256215779](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r256215779) [](ancestors = 256215779)